scripts/init-properties-for-docker.sh

```diff
@@ -17,11 +17,11 @@ if [ -f $resourcefile ]; then
 fi
 
 cat > $resourcefile <<EOF
-ctia.store.es.default.host=$ip
+ctia.store.es.default.host="$ip"
```

Comment:
so back to previous version :).

The `uri` doesn't exists anymore, this is why I had to make this change, to use `host` instead.